standard/sections/clause_7_normative_text.adoc

```diff
@@ -36,7 +36,7 @@ The table below provides an overview of the set of properties that may be includ
 
 |``geometry``
 |**required**
-|Geospatial location associated with the data or metadata, in a geographic coordinate reference system (see <<_geometry>>)
+|Geospatial location associated with the data or metadata (see <<_geometry>>)
```

Comment:
there isn't a CRS property in the geometry section. Alternatively, we can not delete it here and update the geometry section.